src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.cs

```diff
@@ -152,7 +152,7 @@ internal void ValidateCanBeUsedForDeserialization()
         /// TypeInfo (for the cases mentioned above). In addition, methods that have a JsonPropertyInfo argument would also likely
```

Comment:
FWIW I suspect the justification presented in the comments is obsolete:

> The JsonPropertyInfo for this JsonTypeInfo. It is used to obtain the converter for the TypeInfo.

Since all construction paths now require `JsonConverter` it seems like this could be replaced with a `JsonConverter` property.